sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.aest.controller.ts

```diff
@@ -5,7 +5,10 @@ import { AuthorizedParties } from "../../auth/authorized-parties.enum";
 import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
 import BaseController from "../BaseController";
 import { ClientTypeBaseRoute } from "../../types";
-import { ScholasticStandingSubmittedDetailsAPIOutDTO } from "./models/student-scholastic-standings.dto";
+import {
+  ScholasticStandingSubmittedDetailsAPIOutDTO,
+  ScholasticStandingSummaryDetailsAPIOutDTO,
+} from "./models/student-scholastic-standings.dto";
```

Comment:
Can it be imported from an index file?

Reply:
I see that we are not using index for the models which are inside the `route-controllers` directory. Only the exports are made from the controller and the service files inside the index. Do you still want me to change it?

Reply:
The comment was added to the wrong line. It is actually about the below import.
```ts
import { ScholasticStandingControllerService } from "./student-scholastic-standings.controller.service";
```